components/board/upboard/board.go

```diff
@@ -0,0 +1,33 @@
+// Package upboard implements an Intel based board.
```

Comment:
I imagine there are non-Up boards with Intel chips out there. I think this should be just for boards manufactured by Up Board. (I'm open to being wrong about this. Leave as-is if you disagree!)